src/components/molecules/button/index.js

```diff
@@ -2,24 +2,40 @@ import { LitElement, html, css, unsafeCSS } from 'lit-element';
 import { classMap } from 'lit-html/directives/class-map';
 import buttonCSS from './index.scss';
 
+const icon = () => html`
```

Comment:
why not `import` our new icon? and is it ok for @jojackpot to have a different arrow?